app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,50 @@
+<resources>
+    <string name="app_name">FastLyrics</string>
+
+    <!-- Nav drawer -->
+
+    <string name="menu_lyrics">Testo</string>
+    <string name="menu_saved">Salvati</string>
+    <string name="menu_search">Cerca</string>
+    <string name="menu_permission">Permesso per notifiche</string>
+    <string name="menu_settings">Impostazioni</string>
+    <string name="menu_about">Informazioni</string>
+
+    <!-- Menu strings -->
+    <string name="menu_copy_lyrics_to_clipboard">Copia testo negli appunti</string>
+
+    <!-- Lyrics View -->
+    <string name="no_song_playing">Non stai ascoltando niente in questo momento</string>
+    <string name="lyrics_not_found">Impossibile trovare il testo per questa canzone</string>
+    <string name="lyrics_parse_exception">Impossibile processare il testo</string>
+    <string name="lyrics_network_exception">C\'è stato un problema di rete mentre si stava caricando il testo</string>
+    <string name="lyrics_unknown_error">C\'è stato un errore sconosciuto</string>
+
+    <string name="lyrics_provider_over_line">Testo da</string>
+
+    <string name="lyrics_clipboard_label">Testo</string>
+
+    <!-- Saved songs view -->
+    <string name="delete">Elimina</string>
+    <string name="items_selected">%1$d selezionati</string>
+
+    <!-- Notification permissions -->
+    <string name="permission_description">FastLyrics ha bisogno dell'accesso alle notifiche del tuo dispositivo per scoprire qual è il brano in riproduzione. Puoi concedere il permesso nelle impostazioni cliccando sul bottone sottostante</string>
```

Comment:
The apostrophe here has to be escaped: `dell'accesso`. Like this: `dell\'accesso`